src/main/java/com/doongji/nestalk/controller/v1/profile/ProfileRestController.java

```diff
@@ -0,0 +1,39 @@
+package com.doongji.nestalk.controller.v1.profile;
+
+import com.doongji.nestalk.controller.v1.profile.dto.ProfileDto;
+import com.doongji.nestalk.controller.v1.profile.dto.ProfileRequest;
+import com.doongji.nestalk.entity.profile.Profile;
+import com.doongji.nestalk.security.JwtAuthentication;
+import com.doongji.nestalk.service.profile.ProfileService;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.annotation.AuthenticationPrincipal;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@Api(tags = "프로필 APIs")
+@RequestMapping("api/v1")
+@RequiredArgsConstructor
+@RestController
+public class ProfileRestController {
+
+    private final ProfileService profileService;
+
+    @ApiOperation(value = "프로필 등록 (JWT 필요)")
+    @PostMapping(path = "profile/create")
```

Comment:
```suggestion
    @PostMapping(path = "profile")
```

POST 메소드를 사용하기 때문에 create라는 url 경로를 쓰지 않아도 될 것 같습니다.